mods/engine/bs_drops/init.lua

```diff
@@ -46,3 +46,16 @@ core.register_on_punchplayer(function(player, hitter, time_from_last_punch, tool
 end)
 
 minetest.register_on_leaveplayer(dropondie.drop_all)
+
+--[[
+if bs.spectator[Name(hitter)] or bs.spectator[Name(player)] then -- Dont allow spectators do damage.
+		return true
+	end
+]]--
+
+minetest.register_on_item_pickup(
+	function(itemstack, picker, pointed_thing, time_from_last_punch)
+		if bs.spectator[Name(picker)] or bs.spectator[Name(picker)] then
+			return Itemstack("")
```

Comment:
You're gonna wanna check and make sure I used this correctly, its been 3 months since I wrote minetest lua :)